website/source/api/acl/acl.html.md

```diff
@@ -285,6 +285,12 @@ The table below shows this endpoint's support for
 | ---------------- | ----------------- | ------------- | ------------ |
 | `NO`             | `none`            | `none`        | `none`       |
 
+-> **Note** - [ACL token
+replication](/docs/agent/options.html#acl_enable_token_replication) must be
+enabled in any secondary datacenter before login can be used there. Login
+require the ability to create local tokens which is restricted to the primary
```

Comment:
```suggestion
requires the ability to create local tokens which is restricted to the primary
```

Reply:
Not really. You can already create local tokens yourself. The tokens created by login are currently restricted to _only_ be local tokens.